### DISCUSSION TITLE
`next dev` is slow and hogs cpu

### DISCUSSION THREAD

**Discussion by MrDiamondDog** [vercel/next.js#63068] [Help]
### Summary

I am using nextjs for a full stack app im making, and so far `next dev` has been a big bottleneck. I find myself constantly having to wait >10 seconds for a file to save while it’s running on my desktop, and much longer on my laptop. My laptop is lower-end and constantly overheats while running the command. My desktop has an i7 and 40gb of 40gb of ram, so I’m really surprised to see how slow it runs.

### Additional information

```js
Windows 11/Ubuntu
Latest nextjs
```

**Comment by fellipefreiire** [comment]
this is the slowest i've ever seen any framework perform in my entire life, i just can't belive this happens
